concrete/src/User/User.php

```diff
@@ -486,7 +486,7 @@ public function unloadAuthenticationTypes()
     public function logout($hard = true)
     {
         $app = Application::getFacadeApplication();
-        $events = $app['director'];
+        $events = $app->make('director');
```

Comment:
This is a minor change: it simply uses the same way to create something with `$app` as the other lines in the same method.